pkg/controller/eventing/subscription/reconcile.go

```diff
@@ -65,8 +74,17 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	// Reconcile this copy of the Subscription and then write back any status
 	// updates regardless of whether the reconcile error out.
 	err = r.reconcile(subscription)
+	if err != nil {
+		glog.Warningf("Error reconciling Subscription: %v", err)
+		r.recorder.Eventf(subscription, corev1.EventTypeWarning, subscriptionReconcileFailed, "Failed to reconcile Subscription: %v", err)
```

Comment:
Same as above, I think this can be removed because any error that gets here should already have an event emitted.

Reply:
Sounds good. Yes, we can take these ones out. Should I leave the success case though? Or you prefer it to be within the reconcile method instead?

Reply:
I think the success case is fine to keep.

Reply:
Will leave the else here and emit the success event there. 
Will just log in the if condition...